front/tests/utils/scrollHelper.ts

```diff
@@ -4,42 +4,50 @@ interface ScrollOptions {
   stepSize?: number;
   timeout?: number;
 }
-
+/**
+ * Scrolls a specified container element horizontally by the given step size, with a delay between steps.
+ *
+ * @param {Page} page - The Playwright page object.
+ * @param {string} containerSelector - The CSS selector for the scrollable container element.
+ * @param {ScrollOptions} [options={}] - Optional scroll configuration including step size and timeout.
+ * @returns {Promise<void>} - Resolves once the container has been fully scrolled.
+ */
 const scrollContainer = async (
   page: Page,
   containerSelector: string,
   { stepSize = 300, timeout = 20 }: ScrollOptions = {}
 ): Promise<void> => {
-  // Locate the scrollable container once
+  // Locate the scrollable container on the page
   const container = await page.evaluateHandle(
     (selector: string) => document.querySelector(selector),
     containerSelector
   );
-  // Ensure the container exists and has scrollable content
+
+  // Get the scrollable width and visible width of the container
   const { scrollWidth, clientWidth } = await page.evaluate(
     (containerElement) =>
       containerElement
         ? {
             scrollWidth: containerElement.scrollWidth,
             clientWidth: containerElement.clientWidth,
           }
-        : { scrollWidth: 0, clientWidth: 0 },
+        : { scrollWidth: 0, clientWidth: 0 }, // Default if no container found
     container
   );
 
-  // Exit early if there is no significant scrollbar
+  // Exit early if there's little or no scrollable content
   if (scrollWidth <= clientWidth + 200) {
```

Comment:
seems like a magic number, and the reason fo this dpecific size isn't easy to understand, could you create a variable?